commit 65b01ed12720de7f5041aa7e1b2a7f17f8102800
Author: pchunt <peter.hunt@opengear.com>
Date:   Sat Mar 31 16:47:44 2018 -0600

    OG-5735 ModemManager: fix 3gpp2/CDMA SMS UTF-8 translation
    
    - Fix unicode character in string literal "ISO-8859-1" in ModemManager

Index: ModemManager/src/mm-sms-part-cdma.c
===================================================================
--- ModemManager.orig/src/mm-sms-part-cdma.c
+++ ModemManager/src/mm-sms-part-cdma.c
@@ -875,7 +875,7 @@ read_bearer_data_user_data (MMSmsPart
         }
         latin[i] = '\0';
 
-        text = g_convert (latin, -1, "UTF-8", "ISO−8859−1", NULL, NULL, NULL);
+        text = g_convert (latin, -1, "UTF-8", "ISO-8859-1", NULL, NULL, NULL);
         if (!text) {
             mm_obj_dbg (log_object, "            text/data: ignored (latin to UTF-8 conversion error)");
         } else {
